docs/metal_facilities_get.md

```diff
@@ -20,7 +20,9 @@ metal facilities get [flags]
 ### Options
 
 ```
-  -h, --help   help for get
+  -h, --help                     help for get
```

Comment:
It looks like this leaked in from an older version of the facilities subcommand PR